server/src/routes/user.ts

```diff
@@ -0,0 +1,68 @@
+import { PrismaClient, UserRole } from "@prisma/client";
+import express from "express";
+
+export const userRoutes = express.Router();
+const prisma = new PrismaClient();
+
+// Filter using query string in url with parameters role and category
+userRoutes.route("/").get(async (req, res) => {
+  let categoryFilter = (req.query.category as string) || null;
+  let roleFilter = (req.query.role as string) || null;
+  let filter: any = {};
+
+  if (categoryFilter !== null) {
+    const categoryId = parseInt(categoryFilter);
+    if (isNaN(categoryId)) {
+      res.status(400).send("Invalid category query parameter, must be an integer");
+      return;
+    } else {
+      filter.categoryGroupId = categoryId;
+    }
+  }
+
+  if (roleFilter !== null) {
+    roleFilter = roleFilter.toUpperCase();
+    if (!(roleFilter in UserRole)) {
+      res.status(400).send("Invalid user role");
+      return;
+    } else {
+      filter.role = roleFilter;
+    }
+  }
+  let users = await prisma.user.findMany({
+    where: filter,
+  });
+
+  let userUuids: string[] = [];
+  users.forEach(user => userUuids.push(user.uuid));
+  res.status(200).send(userUuids);
```

Comment:
For this endpoint, we want to return everything except the `token` field. Can you update it to reflect that? Also, can you update this endpoint and all the next endpoints to use `.json` instead of `.send`? This adds that the response is of type `application/json`